tests/conftest.py

```diff
@@ -533,7 +533,7 @@ def create_dagrun(self, **kwargs):
 
             if "run_type" not in kwargs:
                 kwargs["run_type"] = DagRunType.from_run_id(kwargs["run_id"])
-            if "execution_date" not in kwargs:
+            if "execution_date" not in kwargs or kwargs["execution_date"] is None:
```

Comment:
Updated the dag_maker.create_dagrun for ease of use

Reply:
Maybe

```suggestion
            if kwargs.get("execution_date") is None:
```